cookiecutter.json

```diff
@@ -5,5 +5,5 @@
     "author_email": "code@inmanta.com",
     "license": "ASL 2.0",
     "copyright": "{% now 'local', '%Y' %} Inmanta",
-    "minimal_compiler_version": "2019.3"
+    "_extensions": ["cookiecutter.extensions.SlugifyExtension"]
```

Comment:
See https://cookiecutter.readthedocs.io/en/latest/advanced/template_extensions.html#slugify-extension. Used to convert `_` to `-` where appropriate (Python package name mostly). We could also define a [custom extension](https://github.com/cookiecutter/cookiecutter/pull/1340) if anyone prefers.